### PR TITLE
feat: add professional partners section

### PR DESCRIPTION
**Pull request by KrissDrawing**
# Pull Request Template

## Issue Overview

This PR addresses #257

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Adds Professional partners tab with 2 new pages.

## How Can This Be Tested/Reviewed?

To show section in navigation you need to add `SHOW_PROFESSIONAL_PARTNERS=TRUE` to .env.local.
Then it will show on right side of navigation. 
There should be 2 pages with content from [here](https://docs.google.com/document/d/12FEQPqAgH3t96IviNgPxox-Bj3j0IqkCdfWYa7h_MYs/edit)

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/public/src/pages/professional-partners/developers-and-property-managers.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  useEffect(() => {
+    pushGtmEvent<PageView>({
+      event: "pageView",
+      pageTitle: "developersAndPropertyManagers",
+      status: profile ? UserStatus.LoggedIn : UserStatus.NotLoggedIn,
+    })
+  }, [profile])
+
+  const metaDescription = t("pageDescription.developersAndPropertyManagers")
```
Comment (ludtkemorgan) [inline sites/public/src/pages/professional-partners/developers-and-property-managers.tsx:21]: This translation key needs to get added to the translation files

File: sites/public/src/tsx_content/professional-partners-developers-cards.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            </li>
+            <li>
+              {t("professionalPartners.dev.howDoWeListPropertyResp3a")}
+              <a href={`mailto:${t("professionalPartners.doorwayEmail")}`}>
+                {t("professionalPartners.doorwayEmail")}
+              </a>
+              {t("professionalPartners.dev.howDoWeListPropertyResp3b")}
+            </li>
+            <li>{t("professionalPartners.dev.howDoWeListPropertyResp4")}</li>
+            <ul className="text__medium-normal list-disc ml-5">
```
Comment (ludtkemorgan) [inline sites/public/src/tsx_content/professional-partners-developers-cards.tsx:94]: for accessibility reasons a `<ul>` can't be a child of a `<ul>`. https://dequeuniversity.com/rules/axe/4.7/list?application=axe-linter. I don't know the best way to get an indented list though with this limitation

**Comment by emilyjablonski** [comment]
Screenshot 2023-09-14 at 2 08 00 PM

**Comment by sarahlazarich** [comment]
@KrissDrawing @emilyjablonski we should be sure the links are accessible, but I'm not sure what the best path is.
@eajensenwa do you have any thoughts on this? Is there a relatively quick fix we can make? Would still like to get something to BR next week if possible.

**Review by ludtkemorgan — APPROVED** [review]

**Pull request merged**